File: firstcode/phase gate one/gateOne/creditCardValidation/python/creditcard.py
import validationfunction
	
print("WELCOME TO THE CREDIT CARD VALIDATOR")

flag = ""

while not flag == "no": 
	
	credit_card_number = input("Enter credit card number to verify:  ")
	
	card_data = validationfunction.card(credit_card_number)
	
	print(f"Credit Card Type: {card_data["card_type"]}")
	print(f"Credit Card Number: {card_data["card_number"]}")
	print(f"Credit Card Digit Length: {card_data["card_digit_length"]}")
	print(f"Credit Card Validity Status: {card_data["card_status"]}")
	
	flag = input("Enter \"yes\" to continue or \"no\" to end app:  \n")

File: firstcode/phase gate one/gateOne/creditCardValidation/java/CreditCard.java
import java.util.Scanner;
import java.util.Arrays;
public class CreditCard {
	public static void main(String[] args) {
	
		System.out.println("WELCOME TO THE CREDIT CARD VALIDATOR\n");

	Scanner input = new Scanner(System.in);
	ValidationFunction feedback = new ValidationFunction();
	String cardNumber = "";
	String[] cardData = new String[4];

	String flag = "";
	
	while(!flag.equals("no")) {
	
		System.out.print("Enter your credit card number:  ");
			cardNumber = input.next();

			cardData = feedback.card(cardNumber);
			
			System.out.printf("Credit Card Type: %s %n", cardData[0]);
			System.out.printf("Credit Card Number: %s %n", cardData[1]);
			System.out.printf("Credit Card Digit Length: %s %n", cardData[2]);
			System.out.printf("Credit Card Validity Status: %s %n", cardData[3]);

	
		System.out.print("Enter yes to continue or no to end the app: \n");
			flag = input.next();
	}
}
}